pkg/apis/metal3/v1alpha1/baremetalhost_types.go

```diff
@@ -124,6 +124,9 @@ type BMCDetails struct {
 	// The name of the secret containing the BMC credentials (requires
 	// keys "username" and "password").
 	CredentialsName string `json:"credentialsName"`
+
+	// Whether to validate certificates for Address URL or not
+	VerifyCA bool `json:"verifyCA,omitempty"`
```

Comment:
Declaring the flag this way is going to mean the default is false, and I don't think that's what we want. How about if we call it something like ignoreUnknownCA or something similar so that the default of false gives us the current behavior?

Reply:
Yeah, as suggested in https://github.com/metal3-io/baremetal-operator/issues/286#issuecomment-524879669 I think the flag should be named something like `insecure` or `disable_host_verification`.